docs/reST/ref/math.rst

```diff
@@ -195,6 +195,35 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector2.distance_squared_to ##
 
+   .. method:: move_towards
+
+      | :sl:`returns a vector moved toward the target by a given distance.`
+      | :sg:`move_towards(Vector2, float) -> Vector2`
+
+      Returns a Vector which moves itself to the given Vector at a given
+      distance.
```

Comment:
I thought we talked about this?
`Returns a Vector which moves itself to the given Vector at a given distance.` is awkward phrasing

`Returns a Vector which is moved towards the given Vector by a given distance`?